### PR TITLE
CTOR-1068 : Refresh cloud-aws-transitgateway

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh cloud-aws-transitgateway

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] 25.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-transitgateway.md [hunk 1]
```diff
@@ -5,79 +5,85 @@ title: AWS Transit Gateway
...
-
-Un compte de service (paire d'identifiants *access/secret keys*) est nécessaire
-afin de pouvoir superviser les resources AWS Transit Gateway. Ce compte doit bénéficier
-des privilèges suivants :
+### Privilèges AWS
 
-| AWS Privilege                  | Description                                                        |
-|:-------------------------------|:------------------------------------------------------------------ |
-| cloudwatch:getMetricStatistics | Get metrics values from Cloudwatch AWS/TransitGateway namespace    |
+Configurez un compte de service (via une combinaison d'access et secret key) et affectez-lui les privilèges suivants :
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-transitgateway.md:79]:
```suggestion
Configurez un compte de service (via une combinaison d'access key et de secret key) et affectez-lui les privilèges suivants :
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-transitgateway.md [hunk 2]
```diff
@@ -90,93 +96,192 @@ yum install perl-Paws
...
-    --proxyurl='http://myproxy.mycompany.org:8080' \
-    --timeframe='600' \
-    --period='60' \
-    --filter-gateway='tgw-01234567890abcd' \
-    --warning-packets-drop-blackhole='500' \
-    --critical-packets-drop-blackhole='1000' \
-    --verbose
+	--plugin=cloud::aws::transitgateway::plugin \
+	--mode=traffic \
+	--custommode='' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-transitgateway.md:259]: pas de custom mode?

File: pp/integrations/plugin-packs/procedures/cloud-aws-transitgateway.md [hunk 3]
```diff
@@ -88,93 +95,192 @@ yum install perl-Paws
...
-    --proxyurl='http://myproxy.mycompany.org:8080' \
-    --timeframe='600' \
-    --period='60' \
-    --filter-gateway='tgw-01234567890abcd' \
-    --warning-packets-drop-blackhole='500' \
-    --critical-packets-drop-blackhole='1000' \
-    --verbose
+	--plugin=cloud::aws::transitgateway::plugin \
+	--mode=traffic \
+	--custommode='' \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-aws-transitgateway.md:258]: custom mode?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-transitgateway.md (hunk 2 above)
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-transitgateway.md:259]:
```suggestion
	--custommode='awscli' \
```

File: pp/integrations/plugin-packs/procedures/cloud-aws-transitgateway.md [hunk 4]
```diff
@@ -88,93 +95,192 @@ yum install perl-Paws
...
-* Select the *Cloud-Aws-Transitgateway-custom* template to apply to the Host.
-* Once the template applied, some Macros marked as 'Mandatory' hereafter have to be configured:
+| Macro         | Description                                                                                                                              | Default value     | Mandatory |
+|:--------------|:-----------------------------------------------------------------------------------------------------------------------------------------|:------------------|:---------:|
+| AWSACCESSKEY  | Set AWS access key                                                                                                                       |                   |     X     |
+| AWSASSUMEROLE | Set Amazon Resource Name of the role to be assumed                                                                                       |                   |           |
+| AWSCUSTOMMODE | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option               |                   |           |
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/cloud-aws-transitgateway.md:207]:
```suggestion
| AWSCUSTOMMODE | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option               |                   | X          |
```

File: pp/integrations/plugin-packs/procedures/cloud-aws-transitgateway.md (hunk 3 above)
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/cloud-aws-transitgateway.md:258]:
```suggestion
	--custommode='awscli' \
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**